### PR TITLE
Added Claimable contract as an extension of Ownable

### PR DESCRIPTION
**Pull request by AugustoL**
New contract template called Claimable, is an extension of Owner contract that overrides the transfer function and obligate the new owner to claim the ownership to complete the transfer of the ownership on the contract.

### CODE REVIEW

**Comment by federicobond** [comment]
Tests could be improved by using new instances of the contract instead of relying on a single deployed copy.

**Comment by maraoz** [comment]
Yes, I mentioned the same thing to @AugustoL yesterday at the meetup. This is a more general problem (other previous tests do this too), so I'll create an issue to refactor tests later: https://github.com/OpenZeppelin/zeppelin-solidity/issues/55

File: contracts/Claimable.sol [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  address public pendingOwner;
+
+  modifier onlyPendingOwner() {
+    if (msg.sender == pendingOwner)
+      _;
+  }
+
+  function transfer(address newOwner) onlyOwner {
+    pendingOwner = newOwner;
+  }
```
Comment (maraoz) [inline contracts/Claimable.sol]: why not directly set owner here, like in Ownable?
I don't see the advantage of having a 2-step change of ownership.

**Comment by maraoz** [comment]
Solves https://github.com/OpenZeppelin/zeppelin-solidity/issues/7

**Pull request merged**